IBPSA/Fluid/FixedResistances/Examples/PlugFlowPipe.mo

```diff
@@ -30,40 +29,90 @@ model PlugFlowPipe "Simple example of plug flow pipe"
     rhoPip=8000,
     T_start_in=323.15,
     T_start_out=323.15) "Pipe"
-    annotation (Placement(transformation(extent={{0,-10},{20,10}})));
-  Modelica.Thermal.HeatTransfer.Sources.FixedTemperature bou(T=283.15)
+    annotation (Placement(transformation(extent={{0,10},{20,30}})));
+  Modelica.Thermal.HeatTransfer.Sources.FixedTemperature bou[2](each T=283.15)
     "Boundary temperature"
     annotation (Placement(transformation(extent={{-40,60},{-20,80}})));
   IBPSA.Fluid.Sources.MassFlowSource_T sou(
-    nPorts=1,
     redeclare package Medium = Medium,
     use_T_in=true,
-    m_flow=3) "Flow source"
-    annotation (Placement(transformation(extent={{-60,-10},{-40,10}})));
+    m_flow=3,
+    nPorts=1) "Flow source"
+    annotation (Placement(transformation(extent={{-60,10},{-40,30}})));
 
   IBPSA.Fluid.Sensors.TemperatureTwoPort senTemOut(
     redeclare package Medium = Medium,
     m_flow_nominal=1,
     T_start=323.15) "Temperature sensor"
-    annotation (Placement(transformation(extent={{30,-10},{50,10}})));
+    annotation (Placement(transformation(extent={{40,10},{60,30}})));
   IBPSA.Fluid.Sensors.TemperatureTwoPort senTemIn(
     redeclare package Medium = Medium,
     m_flow_nominal=1,
     T_start=323.15) "Temperature sensor"
-    annotation (Placement(transformation(extent={{-30,-10},{-10,10}})));
+    annotation (Placement(transformation(extent={{-30,10},{-10,30}})));
+  Sensors.TemperatureTwoPort senTemInNoMix(
+    redeclare package Medium = Medium,
+    m_flow_nominal=1,
+    T_start=323.15) "Temperature sensor"
+    annotation (Placement(transformation(extent={{-30,-30},{-10,-10}})));
+  IBPSA.Fluid.FixedResistances.PlugFlowPipe pipNoMix(
+    have_pipCap=false,
+    redeclare package Medium = Medium,
+    dh=0.1,
+    length=100,
+    dIns=0.05,
+    kIns=0.028,
+    m_flow_nominal=1,
+    cPip=500,
+    thickness=0.0032,
+    initDelay=true,
+    m_flow_start=1,
+    rhoPip=8000,
+    T_start_in=323.15,
+    T_start_out=323.15) "Pipe"
+    annotation (Placement(transformation(extent={{0,-30},{20,-10}})));
+  Sensors.TemperatureTwoPort senTemOutNoMix(
+    redeclare package Medium = Medium,
+    m_flow_nominal=1,
```

Comment:
tau=0?

Reply:
Same for all other examples to be honest: the filter slows down the model and skews the results.